runtime/ftplugin/qf.vim

```diff
@@ -10,7 +10,9 @@ endif
 " Don't load another plugin for this buffer
 let b:did_ftplugin = 1
 
-let b:undo_ftplugin = "set stl<"
+if get(g:, 'qf_default_statusline')
```

Comment:
Please revert the condition, so that the default does not change:

    if !get(g:, 'qf_disable_statusline')

Also, please add a short paragraph at `syntax.txt` describing the variable `:h ft-qf-syntax`.

Reply:
Thank you for comment, I updated the patch.